src/app/game/Ludo/page.tsx

```
@@ -0,0 +1,322 @@
+              key={`${player}-${pawnIndex}`}
+              className={`${styles.pawn} ${styles[player]} ${
+                gameState.movablePawns.includes(pawnIndex) && gameState.currentPlayer === player
+                  ? styles.movablePawn
+                  : ''
+              }`}
+              onClick={() => gameState.currentPlayer === player && movePawn(pawnIndex)}
+              style={{
+                transform: `translate(${(i % 2) * 8}px, ${Math.floor(i / 2) * 8}px)`,
+                zIndex: 10 + i
+              }}
+            />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)